src/components/radios/_radios.scss

```diff
@@ -8,14 +8,16 @@
 
     position: relative;
 
+    min-height: 40px;
```

Comment:
Non-blocking: I see 40px referenced in a few places, should they be a shared variable?

Reply:
Good call. Fixed.